keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md

```diff
@@ -127,7 +127,7 @@ These use cases are out of scope entirely:
 
 This level of the Cluster Management API describes the global configuration of a cluster. It should be capable of representing the versioning and configuration of the entire control plane, irrespective of the representation of nodes.
 
-Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
+Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/tree/release-1.12/cmd/kubeadm/app/apis/kubeadm/v1alpha3) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
```

Comment:
Can we not link to versioned trees?  Let's find where the file actually lives in master

Reply:
the particular API was removed with the release of 1.15.

Reply:
So why not link to the last rev of it?  Or remove the link?

This seems like a never-ending problem - files in the tree move or go away, and these KEPs consume review bandwidth for REALLy no value.

Reply:
removing the link seems best.

@timyinshi 
you can remove the link defined as:

```
`[an existing API]((https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go))
```

and change that to just:
```
an existing API
```